core/node/rpc/notification_test.go

```diff
@@ -297,6 +303,57 @@ func testGDMReactionMessage(
 	}, 5*time.Second, 100*time.Millisecond, "Received unexpected notifications")
 }
 
+// User A, B and C in GDM
+// User A in GDM posts a message
+// User B reacts to user A
+// User A should get a reaction notification, but user C should not
+func testGDMTipMessage(
+	ctx context.Context,
+	test *gdmChannelNotificationsTestContext,
+	nc *notificationCapture,
+) {
+	userA := test.members[0]
+	userB := test.members[1]
+	userC := test.members[2]
+
+	test.subscribeWebPush(ctx, userA)
+	test.subscribeWebPush(ctx, userB)
+	test.subscribeWebPush(ctx, userC)
+
+	messageEvent := test.sendMessageWithTags(ctx, userB, "hi!", nil)
```

Comment:
Should this not be userA instead of userB?

Reply:
Yes. Sorry this is still wip

Reply:
this pr has been updated